app/deploy-cert-to-lb

```diff
@@ -14,18 +14,20 @@ if [[ $DRY_RUN =~ [yY] ]]; then
   exit 0
 fi
 
-oci lb certificate create --load-balancer-id ${LB_OCID} \
+oci_defaults=()
+oci_defaults+=(--load-balancer-id ${LB_OCID})
+
+oci lb certificate create ${oci_defaults[@]} \
   --certificate-name ${CERT_NAME} \
-  --ca-certificate-file ${CERT_PATH}/chain.pem \
   --private-key-file ${CERT_PATH}/privkey.pem \
-  --public-certificate-file ${CERT_PATH}/cert.pem \
+  --public-certificate-file ${CERT_PATH}/fullchain.pem \
```

Comment:
I contemplated using the `fullchain.pem`, however, should this be done for the `ca-certificate-file` as oppose to `pubilc-certificate-file` parameter?

Reply:
I must say that not expert on certificates. But if you see Nginx config how use LetsEncrypt cerificate, ssl_certificate directive is that fullchain.pem and chain.pem file isn't used at all. As I understand load balancer is Nginx.
https://www.nginx.com/blog/using-free-ssltls-certificates-from-lets-encrypt-with-nginx/